pkg/apis/kubeflow/validation/validation.go

```diff
@@ -39,8 +39,11 @@ var (
 
 	validRestartPolicies = sets.NewString(
 		string(kubeflow.RestartPolicyNever),
-		string(kubeflow.RestartPolicyOnFailure),
-	)
+		string(kubeflow.RestartPolicyOnFailure))
+
+	validManagedBy = sets.NewString(
```

Comment:
The validation should be similar to https://github.com/kubernetes/kubernetes/blob/b4ce3a55bc7218a6710ff9340dccd403eee5e293/pkg/apis/batch/validation/validation.go#L220

Reply:
There is no point on following my suggestion if the values are fixed. Forget about this for now.